src/net.cpp

```diff
@@ -1326,57 +1326,53 @@ bool CConnman::InactivityCheck(const CNode& node) const
     return false;
 }
 
-bool CConnman::GenerateSelectSet(std::set<SOCKET> &recv_set, std::set<SOCKET> &send_set, std::set<SOCKET> &error_set)
+bool CConnman::GenerateSelectSet(const std::vector<CNode*>& nodes, std::set<SOCKET> &recv_set, std::set<SOCKET> &send_set, std::set<SOCKET> &error_set)
 {
     for (const ListenSocket& hListenSocket : vhListenSocket) {
         recv_set.insert(hListenSocket.socket);
     }
 
-    {
```

Comment:
I deliberately did the white-space change in a separate commit to ease review which would be harder if logic changes are mixed with lots of white-space changes. I extended the commit message though, so that it is easier to answer "how did we end up with those unnecessary braces".

Reply:
Yes, no worries. Since some time now my `.gitconfig` has this set by default to simplify whitespace review (open to suggestions).
```
[diff]
  renames = copies
	colorMoved = dimmed-zebra
	colorMovedWs = allow-indentation-change
```